protocols/tests/test_migration/test_round_trip_reports_5_0_0_to_reports_6_0_0.py

```diff
@@ -0,0 +1,109 @@
+from protocols import reports_6_0_0
+from protocols import reports_5_0_0
+from protocols.tests.test_migration.base_test_migration import TestCaseMigration
+from protocols.migration.migration_reports_6_0_0_to_reports_5_0_0 import MigrateReports600To500
+from protocols.migration.migration_reports_5_0_0_to_reports_6_0_0 import MigrateReports500To600
+
+
+class TestRoundTripMigrateReports500To600(TestCaseMigration):
+    old_model = reports_5_0_0
+    new_model = reports_6_0_0
+
+    def test_migrate_interpretation_request_rd(self):
+        self._check_cancer(fill_nullables=True)
+
+    def test_migrate_interpretation_request_cancer(self):
+        self._check_rd(fill_nullables=True)
+
+    def test_migrate_interpretation_request_rd_with_nulls(self):
+        self._check_cancer(fill_nullables=False)
```

Comment:
Should `test_migrate_interpretation_request_rd_with_nulls` run `self._check_rd` instead of `self._check_cancer`?

Reply:
as above